pkg/datapath/connector/pod-link.go

```diff
@@ -0,0 +1,134 @@
+package connector
+
+import (
+	"fmt"
+
+	"github.com/cilium/cilium/pkg/datapath/link"
+	"github.com/cilium/ebpf"
+
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/vishvananda/netlink"
+
+	"golang.org/x/sys/unix"
+)
+
+// SetupNicInRemoteNs creates a tail call map, renames the netdevice inside
+// the target netns and attaches a BPF program to it.
+// egress is on 0. ingress is on 1.
```

Comment:
Could you extend the comment saying that the actual programs will be loaded after the loader will call `graftDatapath()`?

Reply:
Yew, I will change it, thanks